specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/examples/Services_CheckStatus.json

```diff
@@ -16,14 +16,29 @@
           "diagnostic.echo",
           "diagnostic.error",
           "diagnostic.fastCount",
+          "DataMigration.AzureSqlDbPostMigrationValidationScenarioId",
           "DataMigration.CompleteAzureSqlDbMIMigration",
           "ConnectToSource.SqlServer",
           "GetUserTables.Sql",
+          "DataMigration.SelectSourceForSqlManagedInstanceScenario",
           "ConnectToTarget.AzureSqlDbMI",
           "ConnectToTarget.SqlDb",
-          "Migrate.SqlServer.SqlDb",
+          "ConnectToTarget.SqlServer",
+          "DataMigration.RunAssessment",
+          "DataMigration.SelectTargetServerForSqlManagedInstanceScenario",
           "Migrate.SqlServer.AzureSqlDbMI",
-          "ValidateMigrationInput.SqlServer.AzureSqlDbMI"
+          "Migrate.SqlServer.SqlDb",
+          "DataMigration.PrepareSqlManagedInstanceRestore",
+          "Migrate.SqlServer.SqlServer",
```

Comment:
I am not sure if Migrate.SqlServer.SqlServer is public yet. It is still in preview